src/components/Sidebar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
+  ChartPieIcon,
+  UserGroupIcon,
+  UsersIcon,
+  GlobeAltIcon,
+  InboxInIcon,
+  CalendarIcon,
+  FolderIcon,
+  AdjustmentsIcon,
+  SupportIcon,
+} from '@heroicons/react/solid';
+import { CogIcon } from '@heroicons/react/outline';
+
+const Sidebar = ({ style }: { style: string }) => {
+  return (
+    <div
+      className={`${style} flex-col fixed h-[100%] pt-16 bg-white dark:bg-dark-bg border-r p-4`}
+    >
+      <div className="list-none pr-8">
+        <li className="mb-4 hover:text-primary">
+          <NavLink to={'to'} className="flex flex-row">
```

Comment:
Can you edit this Link to bring the user at this page by adding the **to** = `/admin/super-admin` 
Click the link is breaking the UI